src/main/java/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,47 @@
+package step2;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    private static final String BASIC_DELIMITER = ",|:";
+    private static final String SPECIAL_DELIMITER = "//(.)\n(.*)";
+
+    private static final String NEGATIVE_NUMBER_ERROR = "문자열에 음수가 포함되어 있습니다.";
+
+    private static final Pattern SPECIAL_PATTERN = Pattern.compile(SPECIAL_DELIMITER);
```

Comment:
```suggestion
    private static final String BASIC_DELIMITER = ",|:";
    private static final String SPECIAL_DELIMITER = "//(.)\n(.*)";
    private static final String NEGATIVE_NUMBER_ERROR = "문자열에 음수가 포함되어 있습니다.";
    private static final Pattern SPECIAL_PATTERN = Pattern.compile(SPECIAL_DELIMITER);
```

개행없이 상수끼리 묶어도 좋아보이는데 어떻게 생각하시나요?